docs/examples/NavbarForm.js

```diff
@@ -0,0 +1,21 @@
+const navbarInstance = (
+  <Navbar>
+    <Navbar.Header>
+      <Navbar.Brand>
+        <a href="#">Brand</a>
+      </Navbar.Brand>
+      <Navbar.Toggle />
+    </Navbar.Header>
+    <Navbar.Collapse>
+      <Navbar.Form pullLeft>
+        <div className="form-group">
+          <Input type="text" placeholder="Search"/>
```

Comment:
This looks unconditional: https://github.com/react-bootstrap/react-bootstrap/blob/master/src/InputBase.js#L206

We need to rewrite `<Input>` though... it's a bit much right now.